pkgs/servers/home-assistant/custom-components/benqprojector/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  benqprojector,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "rrooggiieerr";
+  domain = "benqprojector";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = "homeassistant-benqprojector";
+    tag = version;
+    hash = "sha256-iAFmXL10QqudECsS9u9w7KBETzu9aWCg1EBbFR1ff+o=";
+  };
+
+  dependencies = [ benqprojector ];
+
+  dontBuild = true;
+
```

Comment:
```suggestion
```

The default build phase is already a no-op.

Reply:
Removed. I copied this from another `package.nix`; unfortunately commit 86fff9e3bf8f07f563468853c55e02a672ad43dd did not remove `dontBuild = true` from all affected packages.